api/v3/rpc.go

```diff
@@ -124,9 +126,28 @@ func (s *AncestryServer) PostAncestry(ctx context.Context, req *pb.PostAncestryR
 	if err = g.Wait(); err != nil {
 		return nil, newRPCErrorWithClairError(codes.Internal, err)
 	}
+	var scannedLayers []*database.LayerScanResult
+	for _, layerRequest := range req.Layers {
+		scannedLayer := layerMap[layerRequest.Hash]
+		scannedLayers = append(scannedLayers, scannedLayer)
+	}
+	scannedLayers, err = imgpostprocessor.PostProcessImage(scannedLayers)
```

Comment:
@Allda 
Hey,
I see you're passing an array of layers so that PostProcessImage implementors can work with all of them at once. 
What is the use case here? 
What will your planned implementation do with the array of layers?

Thanks

Reply:
@ldelossa I am passing all layers to post-processor because I need to share some information (potential namespaces) in between layers. You can see an example how it is done in this PR:

https://github.com/quay/clair/pull/864/files#diff-e689e8e675327982224ae6633a2dc773